test/powershell/Modules/Microsoft.PowerShell.Utility/Pester.Commands.Cmdlets.Json.Tests.ps1

```diff
@@ -1455,4 +1454,43 @@ Describe "Json Bug fixes"  -Tags "Feature" {
     {
         RunJsonTest $testCase
     }
+
+    It "ConvertFrom-Json can parse an array of PSObjects as a single string." {
+
+        # Create a couple of PSCustomObjects
+        $obj1 = New-Object PSCustomObject
+        Add-Member -InputObject $obj1 -NotePropertyName "objectName" -NotePropertyValue "object1Name"
+        Add-Member -InputObject $obj1 -NotePropertyName "objectValue" -NotePropertyValue "object1Value"
+
+        $obj2 = New-Object PSCustomObject
+        Add-Member -InputObject $obj2 -NotePropertyName "objectName" -NotePropertyValue "object2Name"
+        Add-Member -InputObject $obj2 -NotePropertyName "objectValue" -NotePropertyValue "object2Value"
+
+        # Add them to an array
+        $array = @($obj1, $obj2)
+
+        # Serialize the array to a text file
+        $filePath = Join-Path $TESTDRIVE test.json
+        $array | ConvertTo-Json | Out-File $filePath -Encoding utf8
+
+        # Deserialize the object back
+        # 1) Read the object as a single string
+        $result1 = Get-Content $filePath -Raw | ConvertFrom-JSON
+        $result1.Count | Should be 2
+
+        # 2) Read the object as an array of PSObjects
+        $result2 = Get-Content $filePath | ConvertFrom-JSON
+        $result2.Count | Should be 2
+
+        <#
+        Create in array with an unusual formatting and deserialize it. This array looks like this:
+        [1,
+        2,
+        3]
+        #>
+        $filePath = Join-Path $TESTDRIVE test2.json
```

Comment:
I'm wondering if this (or the above) needs to be done with a file at all. Does the following cover the case?
```
$result = "[1,","2,","3]" | convertfrom-json
$result.count | Should be 3
```